packages/eventual-send/src/local.js

```diff
@@ -13,6 +18,55 @@ const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
  */
 const isObject = val => Object(val) === val;
 
+const BREAKPOINTS = getEnvironmentOption('BREAKPOINTS', '');
+
+/**
+ * @type {Record<string, [string, (number | '*')][]>}
+ */
+// @ts-expect-error doesn't like null __proto__
+const BREAKS = { __proto__: null };
+
+const BP_REGEXP = /(.+?)\.(\w+?|\*)=(\d+?|\*)/;
+
+for (const bp of BREAKPOINTS.split(',')) {
+  if (bp !== '') {
+    const match = BP_REGEXP.exec(bp);
+    if (match === null) {
+      throw Fail`Expected "tagName.methodName=count" ${q(bp)}`;
+    }
+    const [_, tag, verb, countStr] = match;
+    const patts = hasOwn(BREAKS, verb) ? BREAKS[verb] : (BREAKS[verb] = []);
+    patts.push([tag, countStr === '*' ? countStr : +countStr]);
+  }
+}
```

Comment:
`harden(BREAKS)` after this for loop?

Reply:
The data structure previously name `BREAKS` is now the encapsulated `breakpointsTable`. Its internal representation still needs reform, but this has not yet happened. With the current representation, I countdown by mutating the part containing the `count` number in place. So, not yet resolving, but noting to clarify further review.

Reply:
Since the internal breakpoints table is encapsulated, I didn't freeze so I could continue to do this trick.

Separately, to support interactive change of breakpoints, I now make the external representation available from within the debugger. To ease modification, I separately do not freeze it, so it can be modified in place.

@dckc since your suggestion ended in a "?", I'll consider this suggestion done. But please unresolve this conversation if you disagree.